### PR TITLE
SNOW-210042 Added memory usage to pipe status object

### PR DESCRIPTION
**Pull request by sfc-gh-zli**

### CODE REVIEW

File: src/main/java/com/snowflake/kafka/connector/internal/SnowflakeTelemetryPipeStatus.java [hunk 1]
```diff
@@ -40,6 +40,10 @@
   AtomicLong cleanerRestartCount;               // how many times the cleaner restarted
   static final String CLEANER_RESTART_COUNT     = "cleaner_restart_count";
 
+  // Memory usage
+  AtomicLong memoryUsage;               // how many times the cleaner restarted
```
Comment (sfc-gh-bli) [inline src/main/java/com/snowflake/kafka/connector/internal/SnowflakeTelemetryPipeStatus.java:44]: what do you mean?
`how many times the cleaner restarted`

Comment (sfc-gh-zli) [reply src/main/java/com/snowflake/kafka/connector/internal/SnowflakeTelemetryPipeStatus.java]: Sorry copy&paste error

**Review by sfc-gh-bli — APPROVED** [review]
LGTM

**Review by sfc-gh-sshankar — APPROVED** [review]
Approval

**Pull request merged**